plugins/serendipity_event_spartacus/serendipity_event_spartacus.php

```diff
@@ -701,7 +701,6 @@ function &fetchOnline($type, $no_cache = false)
             }
 
             xml_parser_set_option($p, XML_OPTION_CASE_FOLDING, 0);
-            @xml_parser_set_option($this->parser, XML_OPTION_TARGET_ENCODING, LANG_CHARSET);
```

Comment:
Why was this removed?

Reply:
It caused an error because $this->parser does not exist. And it can not exist, was my thought at time of removal.

Reply:
Oh true, probably it needs to use $p instead of $this->parser. Setting the proper target encoding doesn't seem unimportant to me... :D

Reply:
Worked before^^